### DISCUSSION TITLE
📝 Blog article involving Utoipa

### DISCUSSION THREAD

**Discussion by ghost** [juhaku/utoipa#639] [Show and tell]
Hello!

I wanted to share [my blog article](https://identeco.de/en/blog/generating_and_validating_openapi_docs_in_rust/) involving this wonderful crate and how we use it at our company. I hope it helps more people to discover this project in the future. Any feedback is welcome of course.

Best regards!

**Comment by juhaku** [comment]
Man this article is gold. :medal_sports: Great work, it went through all noticeable features in great detail and even mentioned about future ones too. This is worth of sharing.

Thank you for taking the effort of writing this.

**Comment by jayvdb** [comment]
I submitted it to TWIR https://github.com/rust-lang/this-week-in-rust/pull/4465

**Reply by ghost** [reply]
Thank you! I wasn't aware of this project, immediately subscribed.

**Reply by jayvdb** [reply]
It was included - see https://this-week-in-rust.org/blog/2023/06/28/this-week-in-rust-501/

**Reply by juhaku** [reply]
Very nice
